WalletWasabi.Fluent/ViewModels/Wallets/WalletViewModel.cs

```diff
@@ -59,6 +61,13 @@ protected WalletViewModel(Wallet wallet) : base(wallet)
 
 			History = new HistoryViewModel(this, balanceChanged);
 
+			balanceChanged
+				.Subscribe(_ => IsWalletBalanceZero = wallet.Coins.TotalAmount() == Money.Zero)
+				.DisposeWith(Disposables);
+
+			this.WhenAnyValue(x => x.IsWalletBalanceZero, x => x.History.IsTransactionHistoryEmpty)
+				.Subscribe(tuple => IsEmptyWallet = tuple.Item1 && tuple.Item2);
```

Comment:
Not possible to have a not zero balance while the transaction list is empty.
```
this.WhenAnyValue(x => x.History.IsTransactionHistoryEmpty)
				.Subscribe(value => IsEmptyWallet = value);
```
So this is enough I think.

Reply:
Pushed changes.